src/Header/HeaderInput.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+type Props = {
+  createNewTodo: (title: string) => void;
+  inputRef: React.RefObject<HTMLInputElement>
+};
+
+export const HeaderInput: React.FC<Props> = ({
+  createNewTodo,
+  inputRef,
+}) => {
+  const [query, setQuery] = useState('');
+  // двічі рендериться хедер інпут, треба з мемоізацією розібратися
```

Comment:
Remove comments. If you couldn't figure it out yourself please ask questions in fe_chat and tag mentors.